website/project/model.py

```diff
@@ -2549,6 +2551,16 @@ def set_privacy(self, permissions, auth=None, log=True, save=True):
                 if self.embargo_end_date and not self.is_pending_embargo:
                     self.embargo.state = Embargo.REJECTED
                     self.embargo.save()
+            if auth and not skip_mail:
+                mails.queue_mail(
```

Comment:
It looks like an email gets added to the queue every time any user makes a project public. But we only send it for the first one. Is it possible to check if a user hasn't already received this email before adding it to the queue?